spec/features/tour/create_new_tour_stop_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+feature 'Feature: Create a new stop for a tour.' do
+
+  scenario 'visit tour as authenticated user stop page and make a new stop' do
+    user = FactoryGirl.create(:user)
+    sign_in_as user
+    tour = FactoryGirl.create(:tour, user_id: user.id)
```

Comment:
Don't set ids. You could simply create a tour, and that would create a user for you.

```
tour = FactoryGirl.create(:tour)
user = tour.user
```